packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -1664,6 +1674,38 @@ class AccountInternal extends React.PureComponent {
     await this.refetchTransactions();
   };
 
+  onCreateRule = async ids => {
+    let { data } = await runQuery(
+      q('transactions')
+        .filter({ id: { $oneof: ids } })
+        .select('*')
+        .options({ splits: 'grouped' }),
+    );
+    let transactions = ungroupTransactions(data);
+
+    let rule = {
+      stage: null,
+      conditions: [
+        {
+          field: 'imported_payee',
+          op: 'is',
+          value: transactions[0].imported_payee || null,
+          type: 'string',
+        },
+      ],
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this is great! But could I recommend adding another case for when `imported_payee` is missing?

So if `transactions[0].imported_payee` is not set - we can pre-fill the `payee` field with `payee` value.